openapi/external_api.yaml

```diff
@@ -254,6 +254,10 @@ components:
         configurationVersion:
           type: integer
           format: int64
+        failures:
+          type: array
+          items:
+            $ref: '#/components/schemas/failure'
```

Comment:
is it possible that there is going to be an error from a reconciler not related to any component? I think these might be also useful for KEB if they can occur.

Reply:
Hmmm...I think it's impossible, I think every error will be related with some component. 
@tobiscr what do you think?

Reply:
yes that'S possible.. but this will return a error-response (4xx or 5xx response code) as these kinds of error are not cluster specific